### DISCUSSION TITLE
How to change the position of the expanded row apart from the first and last?

### DISCUSSION THREAD

**Discussion by chanurholis** [KevinVandy/material-react-table#46] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/55186337/186453594-45c5e4fc-fad6-47f8-b3ca-7958497600dd.png)

**Comment by KevinVandy** [comment, marked as answer]
Check out the Column Ordering Docs https://www.material-react-table.com/docs/guides/column-ordering#change-the-default-column-order

**Reply by KevinVandy** [reply]
also if you need to see what the column ids are, read here: https://www.material-react-table.com/docs/guides/display-columns#built-in-mrt-display-columns

**Reply by chanurholis** [reply]
thanks for responding, it works.

maybe I will ask a lot of questions regarding the MRT, is it okay?

**Reply by KevinVandy** [reply]
Of course that's fine. You can even join the discord for easier discussions. Just be sure to see if the answer is already in the docs first. There's even a search box in the docs now.
